zenoh/src/net/routing/interceptor/authorization.rs

```diff
@@ -184,25 +184,31 @@ impl PermissionPolicy {
 struct ActionPolicy {
     query: PermissionPolicy,
     put: PermissionPolicy,
+    delete: PermissionPolicy,
     declare_subscriber: PermissionPolicy,
     declare_queryable: PermissionPolicy,
+    reply: PermissionPolicy,
 }
 
 impl ActionPolicy {
     fn action(&self, action: AclMessage) -> &PermissionPolicy {
         match action {
             AclMessage::Query => &self.query,
             AclMessage::Put => &self.put,
+            AclMessage::Delete => &self.delete,
             AclMessage::DeclareSubscriber => &self.declare_subscriber,
             AclMessage::DeclareQueryable => &self.declare_queryable,
+            AclMessage::Reply => &self.reply,
```

Comment:
From a stylistics point of view let's group `Query` and `Reply` together while matching.
Moreover, let's match first the most common cases that are `Put`, `Query`, Reply.